.github/workflows/samples.yml

```diff
@@ -0,0 +1,49 @@
+# This workflow runs a `deno check` against slack sample apps
+name: Samples Integration Type-checking
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  samples:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        sample:
+          - slack-samples/deno-issue-submission
+          - slack-samples/deno-starter-template
+          - slack-samples/deno-blank-template
+          - slack-samples/deno-message-translator
+          - slack-samples/deno-request-time-off
+          - slack-samples/deno-simple-survey
+
+    steps:
+      - name: Setup Deno
+        uses: denoland/setup-deno@v1
+        with:
+          deno-version: v1.x
+
+      - name: Checkout the sdk
+        uses: actions/checkout@v3
+        with:
+          path: ./deno-slack-sdk
+      - name: Checkout the ${{ matrix.sample }} sample
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ matrix.sample }}
+          path: ./sample
+
+      - name: Set imports.deno-slack-sdk/ to ../deno-slack-sdk/src/ in import_map.json
```

Comment:
Opted to use a deno script over patten matching to allign with the way this Action will be implemented in the deno-slack-api